test/test.js

```diff
@@ -101,9 +101,30 @@ describe('bot', function() {
     const preamble = 'Subscribed to ';
     if (!message.startsWith(preamble))
       return false;
+    // There is an exclamation mark at the end; make sure to remove it.
     return message.substring(preamble.length, message.length - 1);
   }
 
+  // Sends a subscriptions message and returns a list of the games.
+  function get_subscriptions(guild, user) {
+    send(guild, user, '<@botuser> subscriptions');
+    let response = getMessage();
+    assert.ok(response.content.startsWith('<@' + user + '>'));
+    let message = response.content.substring(user.length + 3);
+    if (message.startsWith('You are not'))
+      return [];
+    const preamble = 'You are subscribed to ';
+    assert.ok(message.startsWith(preamble));
+    // Parsing the return is made more difficult due to the fancy formatting.
+    // Woo for functional testing.
+    let game_string = message.substring(preamble.length, message.length);
+    let games = game_string.split(/and|,/);
+    // In the case where we have A, B, and C, we end up with
```

Comment:
This would likely be resolved by a slightly smarter regex, str.split(/,? and|,/)